pkg/drivers/ssh/ssh.go

```diff
@@ -102,6 +105,16 @@ func (d *Driver) PreCreateCheck() error {
 		if _, err := os.Stat(d.SSHKey); os.IsNotExist(err) {
 			return fmt.Errorf("SSH key does not exist: %q", d.SSHKey)
 		}
+
+		key, err := ioutil.ReadFile(d.SSHKey)
+		if err != nil {
+			return err
```

Comment:
Would it makes sense to add a message to the error?

Reply:
Not really expecting any I/O errors here, if the file exists.